.github/workflows/aks-deploy-ui.yaml

```diff
@@ -0,0 +1,371 @@
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+        description: "Deploy environment. Can be development, staging, or production."
+      environmentKeyVault:
+        required: true
+        type: string
+        description: "AKS Key vault."
+      repositoryName:
+        required: true
+        type: string
+        description: "GitHub Repository Name."
+      clusterName:
+        required: true
+        type: string
+        description: "AKS Cluster name."
+      clusterResourceGroup:
+        required: true
+        type: string
+        description: "AKS Cluster Resource Group."
+      aksIngressFqdn:
+        required: true
+        type: string
+        description: "FQDN for AKS cluster ingress."
+      dnsResourceGroup:
+        required: true
+        type: string
+        description: "DNS Resource Group."
+      chartsPath:
+        required: true
+        type: string
+        description: "Relative path to Helm charts."
+      dockerFilePath:
+        required: true
+        type: string
+        description: "Relative path to Dockerfile."
+      dockerImageName:
+        required: true
+        type: string
+        description: "Docker image name."
+      dockerImageTag:
+        required: true
+        type: string
+        description: "Docker image tag."
+      maximumReplicas:
+        required: false
+        type: string
+        description: "Maximum number of replicas for the application HPA."
+        default: 4
+      storageAccountName:
+        required: true
+        type: string
+        description: "Storage Account for application information table."
+      appInfoTableName:
+        required: true
+        type: string
+        description: "Application information storage table."
+    secrets:
+      azureCredentials:
+        required: true
+      registryUserName:
+        required: true
+      registryPassword:
+        required: true
+      storageAccountKey:
+        required: true
+
+jobs:
+  AKS_Deploy:
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    timeout-minutes: 20
+    environment: ${{ inputs.environment }}
+    steps:
+      - id: getlatestrelease
+        if: ${{ inputs.environment == 'production'}}
+        # TODO: verify this is the correct action and that it grabs the correct release when one is deleted
```

Comment:
I added this TODO because when I checked the repo to see if anyone else had issues with incorrect tags after one was deleted, I noticed this issue: https://github.com/gregziegan/fetch-latest-release/issues/10